tests/SDKTest.php

```diff
@@ -31,6 +31,10 @@
     'POST:/v1/mock/tests/general/upload:passed',
 ];
 
+const EXTENDED_GENERAL_RESPONSES = [
+    'Download test passed.',
+];
```

Comment:
Why do we need this?

Reply:
I did add a Test for the file download. 
There was no assert for it, so I added it.